.github/workflows/multi-arch-test-build.yml

```diff
@@ -15,12 +15,12 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          - arch: aarch64_cortex-a53
-            target: mvebu-cortexa53
+          - arch: aarch64_generic
+            target: armsr-armv8
```

Comment:
I am quite curious about this change. Maybe I have not follow it carefully, but is there any reference for proposing this change? While looking at https://downloads.openwrt.org/snapshots/packages/aarch64_cortex-a53/packages/ I can not see such change there since all these packages were build yesterday with ``aarch64_cortex-a53``.

Also mvebu-cortexa53 is still there: https://github.com/openwrt/openwrt/tree/9519a625499a963508755b7fc08fb825b3d8cfde/target/linux/mvebu/cortexa53 under mvebu.

Reply:
Interesting finding, thank you for your explanation. Even though, I would rather see to have updated ``aarch64_cortex-a53`` rootfs container, but LGTM